src/nodes/PragmaDirective.js

```diff
@@ -4,8 +4,21 @@ const {
   }
 } = require('prettier');
 
+const semver = require('semver');
+
 const PragmaDirective = {
-  print: ({ node }) => concat(['pragma ', node.name, ' ', node.value, ';'])
+  print: ({ node }) => {
+    // @TODO: remove hack once solidity-parser-antlr is fixed
+    let value = node.value
+      .replace(/([<>=])/g, ' $1')
+      .replace(/< =/g, '<=')
+      .replace(/> =/g, '>=')
+      .trim();
+    if (value.split(' ').length > 1) {
+      value = semver.validRange(value) || node.value;
```

Comment:
not sure if we need to add `|| node.value`; can you add a test that uses that branch?

Reply:
`semver.validRange()` returns `null` if the string given is not a valid semver range.

In this scenario, I thought it would be better to keep the original `node.value` instead of replacing this with an empty string.

I couldn't trigger this since [solidity-parser-antlr](https://github.com/federicobond/solidity-parser-antlr) takes care of throwing an exception, in case there is an unexpected value. But they already are making a mistake by stripping the spaces between ranges, so who knows.

I added a test for this use case.

Reply:
oh, from a test point of view I was thinking more about adding an additional case to the existing `tests/Pragma/Pragma.sol` file

Reply:
@Janther let me know if you want me to fix the tests

Reply:
Hi @mattiaerre,

I haven't forgotten I just couldn't figure out a scenario where [solidity-parser-antlr](https://github.com/federicobond/solidity-parser-antlr) would return an invalid value for range (other than the one that we are fixing in this PR).

That's why instead of an integration test, I provided a unit test in case we decide to go with a different parser or the current parser changes and allows for an invalid version range (not to mention that there might be a combination that I just missed).

Feel free to add tests and use cases here.

Anyway, this is just meant to be a patch until the parser is fixed.
I've been focusing my effort in #116 at the moment. I'm hoping to show some progress by next week.

Reply:
awesome, let's divide and conquer; I'll fix the tests so that you can focus on #116 which I think it is much more important. thanks again @Janther 👍

Reply:
it seems to me that it's practically impossible to cover this branch:

<img width="396" alt="Screen Shot 2019-03-28 at 4 23 11 PM" src="https://user-images.githubusercontent.com/6423261/55199055-d51d4700-5175-11e9-849d-c48629cb2a8c.png">

I'd be inclined to remove `|| node.value` if you don't mind @Janther

Reply:
I'm ok with this :)

Reply:
I'll remove it and merge this PR 👍